src/lightning/pytorch/strategies/fsdp.py

```diff
@@ -148,7 +151,7 @@ def __init__(
         activation_checkpointing: Optional[Union[Type[Module], List[Type[Module]]]] = None,
         activation_checkpointing_policy: Optional["_POLICY"] = None,
         sharding_strategy: "_SHARDING_STRATEGY" = "FULL_SHARD",
-        state_dict_type: Literal["full", "sharded"] = "full",
+        state_dict_type: Literal["full", "sharded"] = "full",  # FIXME: make sharded the default
```

Comment:
Do we want to make this change yes or no?

Pros:
- consistent with fabric
Cons:
- breaking change

Reply:
I prefer "full" as the default in general since it's easier to deal with for most users.